odl/space/npy_ntuples.py

```diff
@@ -1472,7 +1472,9 @@ def npy_weighted_inner(weight):
 
     See Also
     --------
-    NumpyFnConstWeighting, NumpyFnVectorWeighting, NumpyFnMatrixWeighting
+    NumpyFnConstWeighting
+    NumpyFnArrayWeighting
+    NumpyFnMatrixWeighting
```

Comment:
Matrix weighting won't make sense any more for tensors. I think nobody has ever used it. Remove?